code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -30,20 +30,34 @@
 	slot_flags = SLOT_ID
 
 	var/obj/item/weapon/card/id/front_id = null
+	var/obj/item/weapon/card/id/net_id = null
 
 
-/obj/item/weapon/storage/wallet/remove_from_storage(obj/item/W, atom/new_location)
+/obj/item/weapon/storage/wallet/New()
+	..()
+	net_id = new /obj/item/weapon/card/id(src)

```

Comment:
This will cause issues where game code gets the players name from their currently equipped ID card.